src/components/tree/phyloTree/grid.js

```diff
@@ -357,25 +368,29 @@ export const addTemporalSlice = function addTemporalSlice() {
 
   /* the gray region between the root (ish) and the minimum date */
   if (Math.abs(xWindow[0]-rootXPos) > minPxThreshold) { /* don't render anything less than this num of px */
-    this.groups.temporalWindow.append("rect")
-      .attr("x", rightHandTree ? xWindow[0] : 0)
-      .attr("width", rightHandTree ? totalWidth-xWindow[0]: xWindow[0])
-      .attr("y", 0)
+    const xEnd_start = rightHandTree ? totalWidth - xWindow[0] : xWindow[0]; // ending X coordinate of the "start" rectangle
+    const translateX_start = -totalWidth + xEnd_start; // translation distance for starting temporalWindow
+    this.groups.temporalWindowStart
       .attr("height", height)
-      .attr("fill", fill);
+      .attr("width", totalWidth)
+      .attr("fill", fill)
+      .transition('temporalWindowTransition')
+      .attr("transform", `translate(${translateX_start},0)`);
```

Comment:
Cool! Am i right in understanding that on each update, we modify the rect's position via the transform attr, and in-between transforms the transition smooths things?

Reply:
Correct, this is in contrast to the current behavior of creating a new rectangle on every frame.

Alternately, we have the option of fixing the position, and instead modifying the width on each frame. However, modifying position is cheaper to animate than modifying the width.